### PR TITLE
Replace operators with keywords

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by vinnydiehl** [comment]
Why? This is ugly but if there is a true functional difference in the keywords that you're using I'm all ears.

**Comment by vinnydiehl** [comment]
Also, don't commit anything to the `develop` branch, it's not a development copy but rather a copy for multiple branches to work off of simultaneously. 20 commits coming into `develop` making changes to little bits and pieces to a long list of files is a potential massive headache. 

Merging pull requests into `develop` is also a big no-no, I gave you push rights to this repository so that you're able to push branches to the official repo, you can delete your fork and just push all of your work to feature branches in the official repo here, I'll make sure everything ends up neatly tucked into the `develop` branch with your name on it, I promise. :) Just open up a pull request for each feature branch, they only act as a discussion thread and I NEVER use the merge button on GitHub, I do the merge locally on my machine and tag and sign everything cryptographically, tidy up commits, merge related commits, and just general housekeeping to keep everything running smoothly. When I complete the merge and push it to GitHub the website sees your commits and automatically closes the pull request. 

Also for source wide changes like this it'd be smart to open up and issue and propose the change before you spend too much time. Luckily this was probably just a one liner recursive find. :P 

TL;DR: PRs are intended to be discussion threads made for each feature branch in the repo and should never be merged directly.

**Comment by ghost** [comment]
There is no functional difference.
It is only a #define.
But it makes the code much more readable!

**Comment by vinnydiehl** [comment]
Not at all, it's more typing and takes up more space in long lines of complicated code. Symbolic operators stand out better from actual keywords (especially with syntax highlighting but even so without), and any competent programmer can read operators without thinking about the symbols or what word for that would be, making having to read and parse non-standard operators very much less readable.

Please revert this in one commit to develop. If you aren't comfortable doing that I'll take care of it as soon as I get out of this airport.

**Comment by ghost** [comment]
If it is not useful, why is it part of the standard as iso646.h and ciso646 headers?
If you realy think it is useless, please revert it yourself because I only know how to revert using GitHub.
